server/dataProcessing/transformData.ts

```diff
@@ -630,15 +619,20 @@ const transformToGeojson = (data: DataEntry[]): FeatureCollection => {
 
     Object.entries(input).forEach(([key, value]) => {
       if (key === "alertID") {
-        feature.id = value.substring(4);
+        feature.id = String(value).substring(4);
```

Comment:
Why do we do this substring(4)?  (my question is not about Osa's change, but rather what this line has always been about)

Reply:
...I remember having a good reason for it, but I don't remember what! 😀

Let me try to remember, and push up a code comment. (But will do so separately - this PR is good to merge.)